src/token.rs

```diff
@@ -42,6 +42,52 @@ impl Outcome {
     }
 }
 
+///Provides the used rules for this game.
+///Because there are many different rules, SGF requires
+///mandatory names only for a small set of well known rule sets.
+///Note: it's beyond the scope of this specification to give an
+///exact specification of these rule sets.
+///Mandatory names for Go (GM[1]):
+/// "AGA" (rules of the American Go Association)
+/// "GOE" (the Ing rules of Goe)
+/// "Japanese" (the Nihon-Kiin rule set)
+/// "NZ" (New Zealand rules)
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum Rule {
```

Comment:
I think `RuleSet` is a better name for this

Reply:
I agree